Classes/Controller/ModuleController.php

```diff
@@ -413,10 +416,8 @@ public function countAction(ServerRequestInterface $request)
         $connection = GeneralUtility::makeInstance(ConnectionPool::class)
             ->getConnectionForTable($tablename);
         $count = $connection->count('uid', $tablename, []);
-
-        // Send the response
-        $jsonArray = json_encode($count);
-        return new \TYPO3\CMS\Core\Http\JsonResponse($jsonArray);
+        $array = ['count' => $count];
```

Comment:
I see... `JsonResponse` in the core does an `empty` check and thus ignores zero `0` counts... (that's a core bug)

```
$response = new \TYPO3\CMS\Core\Http\JsonResponse();
return $response->setPayload((int)$count);
```
... would be a work-around for that, keeping previous client-side implementation in place.

Reply:
However like this it is easier for ppl to see what to do if they want to return more then one value